lib/register.js

```diff
@@ -49,7 +46,11 @@ async function register(req, res) {
   }
  // aor, contact, sbcAddress, protocol, expires
   const sbcAddress = req.server.hostport;
-  await registrar.add(aor, contact, sbcAddress, protocol, expires);
+  try {
+	await registrar.add(aor, contact, sbcAddress, protocol, expires);
+  } catch (err) {
+	throw Error(err);
```

Comment:
This should probably be `throw err` since err is already an instance of Error.  And if err was a string, we would want it to be `throw new Error(err)`

More importantly however, now that I looked back at my implementation of the `add` method I can see that it never throws an error or rejects the Promise, only ever resolves with a true or false value.  So (sorry about this) but you can remove the try-catch around the call to add (same with call to remove -- does not throw, simply returns true or false